doc/api/cluster.markdown

```diff
@@ -420,11 +420,12 @@ exit, the master may choose not to respawn a worker based on this value.
 * `message` {Object}
 * `sendHandle` {Handle object}
 
-This function is equal to the send methods provided by
```

Comment:
removing this kind of removes the general description of the function, I think it should have a general description too rather than only two separate ones